contract/src/contractStarter.js

```diff
@@ -27,13 +24,61 @@ import {
   IssuerRecordShape,
 } from '@agoric/zoe/src/typeGuards.js';
 import { depositToSeat } from '@agoric/zoe/src/contractSupport/zoeHelpers.js';
+import { AmountShape } from '@agoric/ertp/src/typeGuards.js';
+import { AmountMath } from '@agoric/ertp/src/amountMath.js';
+import { atomicRearrange } from '@agoric/zoe/src/contractSupport/atomicTransfer.js';
+import { BOARD_AUX_PATH_SEGMENT, publish } from './boardAux.js';
 
 /** @template SF @typedef {import('@agoric/zoe/src/zoeService/utils').StartParams<SF>} StartParams<SF> */
 
+const { fromEntries, keys } = Object;
 const { Fail } = assert;
 
-// /** @type {ContractMeta} */
-// const meta = {};
+/** @type {import('./types').ContractMeta} */
+export const meta = harden({
+  customTermsShape: {
+    prices: {
+      startInstance: AmountShape,
+      installBundleID: AmountShape,
+      storageNode: AmountShape,
+      timerService: AmountShape,
+    },
+    board: M.remotable('board'),
+    namesByAddress: M.remotable('namesByAddress'),
+    agoricNames: M.remotable('agoricNames'),
+    priceAuthority: M.remotable('priceAuthority'),
```

Comment:
Perhaps the "give me a promise for when the price changes by x" API is sufficiently resource-intensive to move priceAuthority this to  `LimitedAccess`